ts/src/pro/binance.ts

```diff
@@ -903,7 +903,7 @@ export default class binance extends binanceRest {
         };
         const newTickers = await this.watch (url, messageHash, this.extend (request, params), messageHash, subscribe);
         if (this.newUpdates) {
-            return newTickers;
+            return this.indexBy (newTickers, 'symbol');
```

Comment:
@sc0Vu i think we need to apply this fix in pretty much all watchTickers implementations

Reply:
turns out, we're already doing something similar in the other implementations

```
        if (this.newUpdates) {
            result[ticker['symbol']] = ticker;
        } else {
            result = this.tickers;
        }
```

Reply:
we have a different problem though, we're not filtering them before returning, so we just return every single ticker